examples/jsm/postprocessing/SSRPass.js

```diff
@@ -219,9 +219,9 @@ var SSRPass = function ( { renderer, scene, camera, width, height, selects, enco
 	}
 
 	this.ssrMaterial = new ShaderMaterial( {
-		defines: Object.assign( {
-			MAX_STEP: Math.sqrt( window.innerWidth * window.innerWidth + window.innerHeight * window.innerHeight )
-		}, SSRShader.defines ),
+		defines: Object.assign( {}, SSRShader.defines, {
```

Comment:
Wait! `Object.assign()` has only two parameters. Isn't it used here with three?

Reply:
I think you can have more than one:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Reply:
Ah, okay! Thanks for the clarification!